shared/src/main/scala/StringHelpers.scala

```diff
@@ -54,4 +124,54 @@ object StringHelpers:
       if index == -1 then c
       else mapping((index + 1) % mapping.length)
     }.mkString
+
+  // https://github.com/DennisMitchell/jellylanguage/blob/70c9fd93ab009c05dc396f8cc091f72b212fb188/jelly/interpreter.py#L1055
+  def sss(compressed: String)(using ctx: Context): String =
+    val decompressed = StringBuilder()
+    var integer =
+      NumberHelpers
+        .fromBase(
+          VList.from(
+            compressed.indices.map(x => CODEPAGE.indexOf(compressed(x)))
+          ),
```

Comment:
```suggestion
          VList.from(compressed.map(CODEPAGE.indexOf)),
```

Reply:
When I tried something like that, it gave a compiler error and complained about having a list of int instead of a list of string

Reply:
Should work now